ceph/UI/dashboard.py

```diff
@@ -0,0 +1,66 @@
+from selenium.common.exceptions import TimeoutException, WebDriverException
```

Comment:
Please revisit the imports, I can see same imports in different files. See if those can be added to the base class or file. Please revisit this logic